### DISCUSSION TITLE
Search for docs

### DISCUSSION THREAD

**Discussion by gkiely** [vanilla-extract-css/vanilla-extract#965] [Ideas]
Sometimes I'm looking for a specific keyword and I would like to be able to search for that in the docs to see where it is referenced. Today it was in regards to using `:hover` within a child selector.

**Comment by trm217** [comment]
Absolutely agree, a search is a must.

**Comment by gkiely** [comment]
This has been added, thank you @askoufis.
https://github.com/vanilla-extract-css/vanilla-extract/pull/1006
